src/Saas.Admin/Saas.Admin.Service/appsettings.json

```diff
@@ -1,12 +1,11 @@
 {
-  "AzureAd": {
-    "Instance": "https://login.microsoftonline.com/",
-    "Domain": "saaskit.onmicrosoft.com",
-    "TenantId": "9d68116c-e8ee-4ee6-9ab5-5c7f11b450bb",
-    "ClientId": "f5fa7382-59b3-4b9f-b996-2a48e3b20ce7",
-    "Scopes": "access_as_user",
-    "CallbackPath": "/signin-oidc",
-    "SignedOutCallbackPath": "/signout-callback-oidc"
+  "AzureAdB2C": {
```

Comment:
these should be empty strings

Reply:
Nothing secret here really, agree for final version though, to force users to create their own.